includes/class-reader-activation.php

```diff
@@ -979,6 +1037,14 @@ public static function process_auth_form() {
 			);
 		}
 
+		// reCAPTCHA test.
+		if ( ! empty( $captcha_token ) ) {
```

Comment:
Same captcha check. We shouldn't skip if captcha is enabled.

Reply:
11e2969 should fix this, too.